app/src/main/java/com/findmyrecycling/dto/Product.kt

```diff
@@ -2,7 +2,12 @@ package com.findmyrecycling.dto
 
 import com.google.gson.annotations.SerializedName
 
-data class Product(var product : String,var id : Int = 0, @SerializedName("name")var common : String ) {
```

Comment:
According to the best practices document, if you have a lot of parameters in a constructor it's better to have each parameter be its own line for better readability

Reply:
Which best practices document?  Cite source here.